CIP-0XXX/README.md

```diff
@@ -0,0 +1,540 @@
+ledger - the Cardano Core team restrains itself to not publishing new `cardano-node`
+versions with significant changes across many of its subsystems - just for the
+sake of clarity of where to look if a bug is found.
+
+In this design, a `Mithril` node runs alongside a `cardano-node`, which is connected to
+it via a UNIX socket (`node-to-client`) protocol.  This means an SPO will run
+a `Mithril` node per `cardano-node`.  The `Mithril` node connected to BP can
+also have access to necessary keys for signing purposes. `Mithril` might also
+use the KES agent, as `cardano-node` will in the near future, to securely access
+the KES key.
+
+
+# Path to Active
+
+## Acceptance Criteria
+
+1. A Cardano node implementing the previously described mini-protocols is released for production.
+1. A message producer node (e.g. a Mithril signer) publishing messages to the local Cardano node through mini-protocols is released.
+1. A message subscriber node (e.g. a Mithril aggregator) receiving messages from the local Cardano node is released.
+
+## Implementation Plan
+
+> [!WARNING]  
+> A hard-fork of the Cardano chain may be required if some information, like peer ports for an overlay network, are to be registered by the SPOs on-chain.
+
+* Write a "formal" specification of the protocols along with vectors/conformance checker for protocol's structure and state machine logic.
+* Write an architecture document extending this CIP with more technical details about the implementation.
+* Validate protocol behaviour with all relevant parties (Network and Node teams).
+* Validate performance profile and impact on Cardano network.
+* Implement the Cardano n2n and n2c mini-protocols in the Cardano node.
+* Implement the Cardano n2c mini-protocols in Mithril signer and aggregator nodes.
+
+# Further Reading
+
+### Cardano network
+
+* **The Shelley Networking Protocol**: https://ouroboros-network.cardano.intersectmbo.org/pdfs/network-spec/network-spec.pdf
+* **Introduction to the design of the Data Diffusion and
+Networking for Cardano Shelley**: https://ouroboros-network.cardano.intersectmbo.org/pdfs/network-design/network-design.pdf
+
+### Mithril
+* **Mithril: Stake-based Threshold Multisignatures**: https://iohk.io/en/research/library/papers/mithril-stake-based-threshold-multisignatures/
+* **Mithril Network Architecture**: https://mithril.network/doc/mithril/mithril-network/architecture
+* **Mithril Protocol in depth**: https://mithril.network/doc/mithril/mithril-protocol/protocol
+* **Mithril Certificate Chain in depth**: https://mithril.network/doc/mithril/mithril-protocol/certificates
+* **Fast Bootstrap a Cardano node**: https://mithril.network/doc/manual/getting-started/bootstrap-cardano-node
+* **Run a Mithril Signer node (SPO)**: https://mithril.network/doc/manual/getting-started/run-signer-node/
+* **Mithril Threat Model**: https://mithril.network/doc/mithril/threat-model
+
+# Copyright
```

Comment:
```suggestion
## Copyright
```
(continued)

Reply:
Done in https://github.com/cardano-scaling/CIPs/commit/a3677312b4fa951484ed769f3b69b6e90e93a742